### ISSUE TITLE
[MM] Missing properties in `azure_sdk_references`

### ISSUE THREAD

**Issue by ArcturusZhang** [Azure/autorest.cli#49]
I am generating `api.yaml` file using `autorest.cli` with input file: `azure-rest-api-specs\specification\compute\resource-manager\readme.md`

# Tag in use
`package-2019-07`.

# Cli section added in `readme.md`

``` yaml $(cli)
cli:
  cli-name: compute
  azure-arm: true
  license-header: MICROSOFT_MIT_NO_VERSION
  payload-flattening-threshold: 2
  namespace: azure.mgmt.compute
  package-name: azure-mgmt-compute
  clear-output-folder: false
  debug: true
  disable-azure-cli: true
```

# Problem encountered

In the generated `magic-module-input` files `computediskencryptionset`, we have two sdk_reference 
 `/properties/previousKeys/sourceVault` and `/properties/previousKeys/sourceVault/id` missing.

**Comment by chunyu3** [comment]
PR: https://github.com/Azure/autorest.cli/pull/52

**Comment by ArcturusZhang** [comment]
After this fix #52 I tried to run cli again, but there are still some sdk reference not generated. 
This time is in `computevirtualmachine`, there is a struct [here](https://github.com/Azure/azure-sdk-for-go/blob/6355969b31502e528a7534fbdcd7cad04496a5fd/services/compute/mgmt/2019-07-01/compute/models.go#L13122) called `WinRMConfiguration`. 
In azure_sdk_reference, we have this entry: `'/properties/osProfile/windowsConfiguration/winRM/listeners'`, but all the sub-entries of this are missing. For instance, there should be `'/properties/osProfile/windowsConfiguration/winRM/listeners/protocol'` and `'/properties/osProfile/windowsConfiguration/winRM/listeners/certificateURL'`.
